chart/kubeapps/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubeapps
-version: 1.5.1
+version: 2.0.0
```

Comment:
this is going to be a breaking change for anyone using these in RoleBindings, so bumping the major version as per semver

Reply:
Since the name is not usable right now, can we consider this a bug fix? People is going to be confused if we bump the major version because of this.